src/StickyMessages.jl

```diff
@@ -93,7 +101,7 @@ function showsticky(io, prev_nlines, messages)
     nothing
 end
 
-function Base.push!(sticky::StickyMessages, message::Pair)
+function Base.push!(sticky::StickyMessages, message::Pair; first=true)
```

Comment:
Looking at this again now, I realize that it'd be better to overload `push!` and `pushfirst!`.